agent/xdsv2/testdata/clusters/destination/l4-multi-destination.golden

```diff
@@ -3,13 +3,7 @@
   "resources":  [
     {
       "@type":  "type.googleapis.com/envoy.config.cluster.v3.Cluster",
-      "name":  "null_route_cluster",
```

Comment:
this gets re-added below.  using update flag puts these in order, so it got moved further down.  This is happens throughout the test files.  This should stabilize now that the ports used in the tests should not change.